.github/workflows/release-body.yml

```diff
@@ -1,18 +1,19 @@
-name: Generate release body for driver
+name: Generate release body for published releases
 on:
   release:
     types: [published]
 
-# Checks if any concurrent jobs is running for driver release CI and eventually cancel it.
+# Checks if any concurrent jobs is running for driver release CI and eventually wait for it.
```

Comment:
If 2 subsequent releases are published, let last one wait for the former CI to finish.
This will allow us to avoid failures when eg: 2 releases are made at the same time, like: `0.16.0` and `7.1.0+driver`.